src/utils/constants.ts

```diff
@@ -19,6 +19,11 @@ export const BLOCKS_TIL_CONFIRMED = 6;
 export const COINBASE_MATURITY_DELAY = 100;
 // https://github.com/bitcoin/bitcoin/blob/v0.19.0.1/src/chainparams.cpp#L258
 export const HALVING_INTERVAL = 150;
+export const TARO_MIN_LND_BALANCE = 10000;
+export enum TARO_ASSET_TYPE {
+  NORMAL = 'NORMAL',
+  COLLECTIBLE = 'COLLECTIBLE',
+}
```

Comment:
I think this enum should be provided by the `taroapi` package. If not, it would be better placed in the `lib/taro/types.ts` file instead of here.